dapptools/src/dapp.rs

```diff
@@ -125,11 +140,24 @@ fn main() -> eyre::Result<()> {
         }
         Subcommands::Build {
             opts:
-                BuildOpts { contracts, remappings, remappings_env, lib_paths, out_path, evm_version: _ },
+                BuildOpts {
+                    contracts,
+                    remappings,
+                    remappings_env,
+                    lib_paths,
+                    out_path,
+                    evm_version: _,
+                    contracts_env,
+                },
         } => {
             // build the contracts
             let remappings = utils::merge(remappings, remappings_env);
             let lib_paths = utils::default_path(lib_paths)?;
+            let contracts = if let Some(env_defined_contracts) = contracts_env {
+                env_defined_contracts
+            } else {
+                contracts
+            };
```

Comment:
this should work as well, right?
```suggestion
            let contracts = contracts_env.unwrap_or(contracts);
```